packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -374,7 +375,24 @@ export const NewDialogueCheckNotification = registerNotificationType({
   userSettingField: "notificationNewDialogueChecks",
   allowedChannels: ["onsite", "none"],
   async getMessage(props: GetMessageProps) {
-    return `New users interested in dialoguing with you (not a match yet)`
+    let notificationAbGroup = ""
```

Comment:
You can just do this assignment on line 382

Reply:
(and it can be a const)

Reply:
but what if there's no userId?

Reply:
You're only using `notificationAbGroup` inside if the `if (userId)` block; you'd only do this kind of split assignment if you were going to use it outside of that block.

Reply:
Oh, I misread.  One sec.

Reply:
I'd probably either move the `switch` statement into the `if (userId)` block along with the `notificationAbGroup` assignment, and get rid of the `default` case, or give `notificationAbGroup` a default value of `v1` when you assign it.  (Also would put a new line between the end of the `if` block and the `switch` statement.)